apps/dashboard/package.json

```diff
@@ -41,17 +41,21 @@
     "class-variance-authority": "^0.7.0",
     "clsx": "^2.1.1",
     "date-fns": "^4.1.0",
+    "debounce": "^2.2.0",
```

Comment:
Nit-picky one: Prefer lodash so that we have a single point of reference for such utilities.